src/cli/Create.cpp

```diff
@@ -97,21 +109,24 @@ int Create::execute(const QStringList& arguments)
 
     auto key = QSharedPointer<CompositeKey>::create();
 
-    auto password = Utils::getPasswordFromStdin();
-    if (!password.isNull()) {
-        key->addKey(password);
+    if (parser->isSet(Create::SetPasswordOption)) {
+        auto password = Utils::getPasswordFromStdin();
+        if (!password.isNull()) {
```

Comment:
I prefer to use `QString::isEmpty()`.

Reply:
This is actually a `QSharedPointer<PasswordKey>`, and not a `QString` password. I renamed the variable to make it more obvious.